### PR TITLE
NAD detail page

### PR DESCRIPTION
**Pull request by upalatucci**
Screenshot from 2024-03-04 18-09-33

image

### CODE REVIEW

**Review by avivtur — APPROVED** [review]
Looking good :+1: 
/lgtm

File: src/views/nads/details/tabs/yaml/NADYAMLPage.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  obj?: NetworkAttachmentDefinitionKind;
+};
+
+const NADYAMLPage: FC<NADYAMLPageProps> = ({ obj: nad }) => {
+  const loading = (
+    <Bullseye>
+      <Loading />
+    </Bullseye>
+  );
+  return !nad ? (
```
Comment (hstastna) [inline src/views/nads/details/tabs/yaml/NADYAMLPage.tsx:18]: I prefer to see an extra line above `return` block, to make the code even more readable, but it's nothing so important. Up to you :)

File: src/views/nads/details/tabs/yaml/NADYAMLPage.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    loading
+  ) : (
+    <React.Suspense fallback={loading}>
```
Comment (hstastna) [inline src/views/nads/details/tabs/yaml/NADYAMLPage.tsx:21]: Maybe we could import `Suspense` same as `FC` and other stuff from 'react'?

**Review by avivtur — APPROVED** [review]
/lgtm

**Pull request merged**